File: ShadowFrameworkLite20/src/shadow/renderer/data/SFInstancedReferenceNodeData.java
package shadow.renderer.data;

import shadow.renderer.SFAbstractReferenceNode;
import shadow.renderer.SFInstancedReferenceNode;

public class SFInstancedReferenceNodeData extends SfAbstractReferenceNodeData {

	@Override
	protected SFAbstractReferenceNode generateReferenceNode() {
		return new SFInstancedReferenceNode();
	}
}

File: trunk/ShadowFrameworkJS/src/shadow/renderer/data/SFInstancedReferenceNodeData.js

function SFInstancedReferenceNodeData(){
	this.generate();	
}

inherit(SFInstancedReferenceNodeData,SFAbstractReferenceNodeData);

SFInstancedReferenceNodeData.prototype["generateReferenceNode"]=function(){
	return new SFInstancedReferenceNode();
};

SFInstancedReferenceNodeData.prototype["generateNewDatasetInstance"]=function(){
	return new SFInstancedReferenceNodeData();
};

